src/wrapper/data_chunk.mli

```diff
@@ -7,6 +7,12 @@ val get_exn : t -> 'a Type.Typed_non_null.t -> int -> 'a array
 val get_opt : t -> 'a Type.Typed.t -> int -> 'a option array
 val free : t -> here:Source_code_position.t -> unit
 
+(** Returns the number of columns in the data chunk. *)
+val column_count : int -> t -> int
+
+(** Returns a human-readable string representation of the data chunk *)
+val to_string_hum : ?bars:[ `Ascii | `None | `Unicode ] -> int -> t -> string
```

Comment:
It's non-obvious what the integer argument here is. In general, in these sorts of cases, you should prefer using labelled arguments.

Reply:
I don't think you've addressed this issue.